etc/hotplug.d/ntp/on-time-changed

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/lib/node_modules/@yoda/flora/bin/flora-cli post yodaos.on-time-changed "[\"$ACTION\"]"
```

Comment:
Does this get called often? If yes, we need to write in C/C++.

Reply:
As a transitioning solution, we'd have this cli tool as soon as possible. After then we could rewrite the tool in C/C++ with same cli signature.